book/src/04_4_interrupts.md

```diff
@@ -11,32 +11,36 @@ TODO why are some functions called with some(...), esp!(...) or "normal"?
 
 ## Tasks
 
-1. Configure the button (GPIO 9) with a c struct [`gpio_config_t`](https://esp-rs.github.io/esp-idf-sys/esp_idf_sys/struct.gpio_config_t.html)the following settings:
+✅ Configure the button GPIO 9 with a c struct [`gpio_config_t`](https://esp-rs.github.io/esp-idf-sys/esp_idf_sys/struct.gpio_config_t.html)the following settings (1. in the exercise/src/main.rs):
```

Comment:
Please leave the numbers as they corresponded to what I asked in the code. when the numbers are there, you don't need to reference to the numbers in ( )